controllers/userController.js

```diff
@@ -55,23 +55,18 @@ module.exports = {
       }
 
       const passwordCompare = await bcrypt.compare(password, user.password);
-      // console.log(passwordCompare);
       if (passwordCompare) {
-        const { id: user_id, username: foundedUsername } = user;
-
         let token = crypto
           .createHash("md5")
-          .update(foundedUsername)
+          .update(`${username}${password}${process.env.SECRETKEY}`)
           .digest("hex");
-
-        const exTime = new Date();
-
-        await Access_Token.create({
-          user_id,
-          access_token: token,
-          expiry: exTime,
+        const expiryTime = new Date();
```

Comment:
@akaashinno you still haven't fixed the +1hr add to expire time.

Reply:
@meanstack9716 sir, for this we are using middleware "verifyToken" in which we are checking expiry time i.e., 1hr. If expiry time is less than 1hr then it will go to controller else it will show error message.

Reply:
resolved